api/swagger.yaml

```
@@ -8343,6 +8343,16 @@ paths:
           description: "server error"
           schema:
             $ref: "#/definitions/ErrorResponse"
+      parameters:
+        - name: "types"
+          in: "query"
+          description: |
+            Object types, for which to compute and return data. For example, `containers,images`.
+          type: "array"
```

Remind me to check if this is the right description for a query-parameter that can be passed multiple times

---

https://swagger.io/docs/specification/2-0/describing-parameters/#array

---

Ah, thanks! TIL about `collectionFormat`.

Looks like the current implementation (allowing both multiple params _and_ comma-separated values), cannot be described in Swagger. 

So it would require this to have either `collectionFormat: csv` (single, comma-separated values `?param=foo,bar,baz`):

```suggestion
        - name: "types"
          in: "query"
          description: |
            Object types, for which to compute and return data. For example, `containers,images`.
          type: "array"
          collectionFormat: csv
```


Or require `collectionFormat:multi` (multiple params, each with one option `?param=foo&param=bar&param=baz`):

```suggestion
        - name: "types"
          in: "query"
          description: |
            Object types, for which to compute and return data. For example, `containers,images`.
          type: "array"
          collectionFormat: multi
```